hs_tools_resource/utils.py

```diff
@@ -0,0 +1,26 @@
+from string import Template
+import logging
+
+logger = logging.getLogger(__name__)
+
+def parse_app_url_template(url_template_string, term_dict_list=[]):
```

Comment:
do not use mutable values as defaults in function parameters, as they are shared between *all* calls to the function. Use a tuple instead.